fastembed/embedding.py

```diff
@@ -126,26 +149,37 @@ def onnx_embed(self, documents: List[str]) -> np.ndarray:
             )
 
         model_output = self.model.run(None, onnx_input)
-        last_hidden_state = model_output[0][:, 0]
+        last_hidden_state = self._pooling_fn(model_output[0], attention_mask)
```

Comment:
Since the `pooling_strategy` parameters are specific to Jina, would prefer to have them in the `JinaEmbedding` class. 

The `JinaEmbedding` class can re-implement the `onnx_embed` function as well.